TCAT/AppDelegate.swift

```diff
@@ -106,6 +132,21 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     func applicationWillTerminate(_ application: UIApplication) {
         // Called when the application is about to terminate. Save data if appropriate. See also applicationDidEnterBackground:.
     }
+    
+    func handleShortcut(item: UIApplicationShortcutItem) {
+        let optionsVC = RouteOptionsViewController()
+        if let shortcutData = item.userInfo {
+            let placeData = NSKeyedUnarchiver.unarchiveObject(with: shortcutData["place"] as! Data)
+            guard let destination = placeData as? Place else {
+                return
+            }
+            optionsVC.searchTo = destination
+            
+            if let navController = window?.rootViewController as? UINavigationController{
```

Comment:
nitpick: space at end

Reply:
before you push to master, make sure to run `swiftlint autocorrect` in the terminal! Will fix issues like ^

Reply:
We should add this to the build phase